### ISSUE TITLE
NodeJS-aws-sdk upgrade to 2.68.0 not honoring sigv4 requests for S3

### ISSUE THREAD

**Issue by randylimmm** [aws/aws-sdk-js#3294]
Confirm by changing [ ] to [x] below to ensure that it's a bug:
- [x] I've gone through [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/welcome.html) and [API reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/)
- [x] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-js) for answers
- [x] I've searched for [previous similar issues](https://github.com/aws/aws-sdk-js/issues) and didn't find any solution
  
**Describe the bug**
We have recently upgraded to NodeJS-aws-sdk 2.68.0 in order to comply with S3 signature v4 requests. However we are still seeing sigv4 traffic as noted when inspecting the Request Headers, under Authorization we see signatures with 'AWS' which signify sigv2. However, sigv4 should begin with `AWS4-HMAC-SHA256`. 

**Is the issue in the browser/Node.js?**
Node.js

**If on Node.js, are you running this on AWS Lambda?**
No

**Details of the browser/Node.js version**
Paste output of `npx envinfo --browsers` or `node -v`

**SDK version number**
Example: v2.68.0
* For browsers, the SDK version number is in the script tag <pre>src=".../aws-sdk-<b>2.466.0</b>.min.js"</pre>
* For Node.js, get SDK version by
  * running command `npm list aws-sdk` from your root directory
  * printing the output of `console.log(AWS.VERSION)` in your code where `AWS = require("aws-sdk");`
  * if running on Lambda and using SDK provided by Lambda runtime, you can find the SDK versions [here](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) 

**To Reproduce (observed behavior)**
Steps to reproduce the behavior (please share code or minimal repo)

**Expected behavior**
Be able to send sigv4 requests when uploading to S3 bucket using the support aws-sdk version

**Screenshots**

**Additional context**
I also am able to see `X-Amz-User-Agent: aws-sdk-js/2.68.0 callback` as part of the request header when making ingestions

**Comment by philipwigg** [comment]
Do you actually mean ```v2.68.0```? That's from 2017.

If you did mean that version, can you please upgrade to a much more recent version (say, ```v2.690.0```) and try again?

**Comment by randylimmm** [comment]
Yes, per the aws docs: https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingAWSSDK.html#specify-signature-version. 2.68.0 should be supporting sigv4.

**Comment by philipwigg** [comment]
Sure, but that's the minimum version and very old now.

I recommend trying to upgrade to a much more recent version and seeing if that fixes your issue.

**Comment by ajredniwja** [comment]
Hey @randylimmm I believe the DOC might be outdated in this case, can you try switching to a more recent version as suggested by @philipwigg ?

Thanks
